src/Explorer/Tree/ResourceTree.tsx

```diff
@@ -162,7 +167,11 @@ export const ResourceTree: React.FC<ResourceTreeProps> = ({ container }: Resourc
       myNotebooksContentRoot,
       (item: NotebookContentItem) => {
         container.openNotebook(item).then((hasOpened) => {
-          if (hasOpened) {
+          if (
+            hasOpened &&
+            userContext.features.notebooksTemporarilyDown === false &&
```

Comment:
maybe add a separate helper that combines notebookstemporarilydown + phoenix

Reply:
I think its not required. Just used only at one place.